apps/web/assets/style.scss

```diff
@@ -83,3 +83,15 @@ apple-pay-button {
 .no-preflight * {
   all: revert;
 }
+
+.no-preflight h1 {
+  display: block;
```

Comment:
use scss nesting please

Reply:
nope, this is a special case, nesting will not work because of revert